config/public-cloud-dev.json

```diff
@@ -92,7 +92,7 @@
     }
   },
   "cxKeyVault": {
-    "name": "arohcp-cx-dev-1",
+    "name": "ah-dev-cx-dev-999",
```

Comment:
Ah... is `ah-` `AroHcp-`? If so, almost seems like we can snip off that prefix since presumably everything in the subscription deploying this will be for ARO HCP - so every resource would have the prefix.